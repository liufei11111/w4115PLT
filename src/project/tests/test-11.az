Matrix m1(1,2);
Matrix m2(1,2); 
Matrix m3(1,2);

m1[1][1] = 2;
Float a;
a = m1[1][1];
m2 [1][1] = m1 [1][1];

m3 = m1 +. m2;
m2 = m2 -. m3;
m1 = m2 *. m3;
m3 = m1 /. m2;
