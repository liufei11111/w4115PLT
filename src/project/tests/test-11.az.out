stmts:

matrix m1(1, 2);
matrix m2(1, 2);
matrix m3(1, 2);
m1[1 , 1]  = 2;
int a;
a = m1[1 , 1] ;
m2[1 , 1]  = m1[1 , 1] ;
m3 = m1 +. m2;
m2 = m2 -. m3;
m1 = m2 *. m3;
m3 = m1 /. m2;

fdecls:



